src/integrations/contact-input-custom-chip-component.test.tsx

```diff
@@ -31,6 +41,53 @@ const chevronTestId = 'icon: ChevronDownOutline';
 const selectAll = 'Select address';
 
 describe('Contact input custom chip component', () => {
+	test('if chipDisplayName is not passed it will show chips label by default', () => {
+		setupTest(
+			<ContactInputCustomChipComponent
+				id={user1.id}
+				label={user1.label}
+				email={user1.email}
+				isGroup={false}
+				_onChange={jest.fn()}
+				contactInputValue={[]}
+			/>
+		);
+		const defaultChipLabel = screen.getByText(user1.label);
+
+		expect(defaultChipLabel).toBeInTheDocument();
```

Comment:
prefer toBeVisible, since it also checks that the element is visible. toBeInTheDocument just checks that the element is inside the dom, but it could be hidden to the user